tsconfig.json

```diff
@@ -14,23 +14,26 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
-    "types": ["react", "node"],
+    "types": ["react", "jest", "node"],
     "plugins": [
       {
-        "name": "next"
-      }
+        "name": "next",
+      },
```

Comment:
What is the purpose of trailing comma here?

Reply:
conflict resolution error! There's been a mix-up.